data_store/common.py

```diff
@@ -0,0 +1,17 @@
+from datetime import datetime
+
+# Maps Pathfinder reporting round numbers to observation window start and end dates
+PF_REPORTING_ROUND_NUMBER_TO_OBSERVATION_DATES = {
+    1: {
+        "start": datetime(2024, 1, 1),
+        "end": datetime(2024, 3, 31, 23, 59, 59),
+    },
+}
+
+# Maps Pathfinder reporting round numbers to submission period start and end dates
+PF_REPORTING_ROUND_NUMBER_TO_SUBMISSION_DATES = {
+    1: {
+        "start": None,
+        "end": datetime(2024, 4, 30, 23, 59, 59),
+    },
+}
```

Comment:
ur is this the same as what we have in data_store/const.py below? I am confused at the duplication.

Reply:
My bad, this should have been removed completely - will do that now